Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -1179,52 +1153,58 @@ variable {ι : Type _}
 
 theorem coe_supᵢ_of_directed [Nonempty ι] {S : ι → Subalgebra R A} (dir : Directed (· ≤ ·) S) :
     ↑(supᵢ S) = ⋃ i, (S i : Set A) :=
+  let i := @Nonempty.some ι inferInstance
   let K : Subalgebra R A :=
     { carrier := ⋃ i, S i
-      mul_mem' := fun x y hx hy =>
+      zero_mem' := by simp only [Set.mem_unionᵢ, SetLike.mem_coe]; exact ⟨i, zero_mem _⟩
+      one_mem' := by simp only [Set.mem_unionᵢ, SetLike.mem_coe]; exact ⟨i, one_mem _⟩
```

Comment:
These should have a porting note; the auto_params are supposed to do this for you.

Reply:
Done.